code/API_definitions/check-device-connectivity.yaml

```diff
@@ -172,6 +224,10 @@ components:
       type: array
       items:
         type: string
+    NetworkConnectionEventType:
+      type: string
+      enum:
+        - IS_CONNECTED
```

Comment:
1. Add a description
2. Change the type to ```boolean```
3. Remove the enum
4. (optional) add ```example: true``` (or false) -> Default is "true" so you can leave it also empty